inst/shiny-examples/SCD-effect-sizes/code-chunks/dat-clean.R

```diff
@@ -0,0 +1,10 @@
+# clean data
+library(dplyr)
+
+dat <-
+  dat %>%
+  group_by({user_grouping}) %>%
+  arrange({user_grouping}, {user_session_number}) %>%
```

Comment:
We can skip the arrange() step if we use `{user_session_number}` in `calc_phase_pairs()`.